extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -245,7 +250,11 @@ export class AmpAutocomplete extends AMP.BaseElement {
    */
   createContainer_() {
     const container = this.element.ownerDocument.createElement('div');
+    const viewHeight = this.viewport_.getHeight() || 0;
     container.classList.add('i-amphtml-autocomplete-results');
+    if (this.inputElement_.getBoundingClientRect().top > (viewHeight / 2)) {
```

Comment:
nit: you should refactor this boolean logic into a method like `shouldRenderAbove` to dedupe, in case you want to change the condition heuristic later.